automation/vars/default.yaml

```diff
@@ -45,7 +45,7 @@ vas:
           - >-
             oc -n openstack wait
             osdpd edpm-deployment-post-ceph --for condition=Ready
-            --timeout=1200s
+            --timeout=40m
```

Comment:
In the following we bumped this to 35 minutes. I assume @eurijon does not mind if we add 5 more minutes.

  https://github.com/openstack-k8s-operators/architecture/pull/136/files